openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -657,7 +698,10 @@ def get_template_path(self):
 
         if path and os.path.exists(path):
             self.log.info("Found template at: '{}'".format(path))
-            return path
+            return {
+                "path": path,
+                "placeholder_keep": placeholder_keep
```

Comment:
Rather use `keep_placeholder` then `placeholder_keep` (it sound like like a fortress of placeholders :) ).

Reply:
Can we please rename `placeholder_keep` to `keep_placeholder`? Both are used in code and only one makes sense.